### PR TITLE
teams/tidb: a vote for sylzd as the reviewer

### PR DESCRIPTION
**Pull request by bb7133**
As described in the title, thanks.

### CODE REVIEW

**Review by winkyao — APPROVED** [review]
LGTM

**Review by tiancaiamao — APPROVED** [review]

File: votes/0550-sylzd-as-tidb-reviewer.md
Comment (tisonkun) [inline votes/0550-sylzd-as-tidb-reviewer.md:23]: @bb7133 I think this vote can be concluded with result updated.

Comment (tisonkun) [reply votes/0550-sylzd-as-tidb-reviewer.md:23]: cc @tiancaiamao you can also take over the remaining work.

**Comment by bb7133** [comment]
Hi @pingcap/tidb-maintainers, 3 '+1 votes' have been collected, according to the current rule, I will merge it tomorrow if there's no new '-1 vote'.

**Review by disksing — APPROVED** [review]

**Pull request merged**
